Storage/ThirdParty/SwiftData.swift

```diff
@@ -202,6 +241,12 @@ open class SwiftData {
         }
     }
 
+    public func cancel() {
+        if let c = getSharedConnection() {
```

Comment:
You should probably go a little bit lower-level than this: this call will create one if there's no open connection, and we should never need to do that.

It would also be prudent to make sure we have a handle to interrupt.

Consider

```
self.sharedConnectionQueue.sync {
    if let c = self.sharedConnection {
        guard let db = c.sqliteDB {       // We do this with a nil check elsewhere, but guard let should work.
            sqlite3_interrupt(db)
        }
    }
}
```